guides/plug.md

```diff
@@ -144,21 +144,21 @@ defmodule HelloWeb.Endpoint do
 
 The default endpoint plugs do quite a lot of work. Here they are in order:
 
-- [Plug.Static](https://hexdocs.pm/plug/Plug.Static.html) - serves static assets. Since this plug comes before the logger, serving of static assets is not logged
+- `Plug.Static` - serves static assets. Since this plug comes before the logger, serving of static assets is not logged
 
-- [Phoenix.CodeReloader](https://hexdocs.pm/phoenix/Phoenix.CodeReloader.html) - a plug that enables code reloading for all entries in the web directory. It is configured directly in the Phoenix application
```

Comment:
REVIEWER: I deleted this line here because this Plug it's already included in the list right after _"In the middle of the endpoint, there is also a conditional block:"_, which seems a more appropriate place for this Plug.